docs/porting/security/psa_targets_in_mbed-os.md

```diff
@@ -39,16 +39,6 @@ Example of a single-core Armv7-M PSA target's inheritance:
     }
 ```
 
-### Naming convention for dual-core and Armv8-M targets
-
-As explained above, dual-core and Armv8-M PSA targets require only an NSPE target name. For Armv8-M non-PSA targets, you can define both SPE and NSPE target names.
-
-The naming convention is:
-
-- `TargetName`: PSA non-secure target (NSPE).
-- `TargetName_NPSA_S`: Non-PSA secure target.
-- `TargetName_NPSA`: Non-PSA non-secure target.
-
```

Comment:
So what is the new convention ?

For ex, current NUCLEO_L552ZE_Q is used as a legacy coretex M target without trust zone.
When we will support TFM, how this new target should be called ?

Reply:
> For ex, current NUCLEO_L552ZE_Q is used as a legacy coretex M target without trust zone.
> When we will support TFM, how this new target should be called ?

I think it should still be called NUCLEO_L552ZE_Q if we support only one flavour. Unless the intention is to provide users both TF-M and non-TF-M options?

Reply:
First intention was to provide both options.
But maybe we need to change our mind.

@LMESTM

Reply:
> First intention was to provide both options.

This should be okay, we can still have two target names in this case I think. I removed the naming guideline section from the doc, only because no targets currently follow them.

Reply:
I'd recommend removing the naming guideline for now as it is not currently used. As @LDong-Arm says, both options could still be used and so this could be re-visited when a use case arises.